frontend/app/mocks/node.ts

```diff
@@ -12,7 +11,6 @@ import { getWSAddressApiMockHandlers } from '~/mocks/wsaddress-api.server';
 import { mockEnabled } from '~/utils/env-utils.server';
 
 export const server = setupServer(
-  ...(mockEnabled('application-history') ? getApplicationHistoryApiMockHandlers() : []),
```

Comment:
We should be able to remove `application-history` from the `ENABLED_MOCKS` environment variable (in `.env.example`, `env-utils.server` and GitOps' `frontend/config.conf`)

Reply:
usually there's a badge that says 'outdated' after making the change, I don't see it in the github ui, but I did make the change.